jboss-kie-kieserver/added/launch/jboss-kie-kieserver-hooks.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+source "${JBOSS_HOME}/bin/launch/launch-common.sh"
+source "${JBOSS_HOME}/bin/launch/jboss-kie-common.sh"
+source "${JBOSS_HOME}/bin/launch/logging.sh"
+
+# make sure there is a workbench service within the namespace.
+serviceName=${WORKBENCH_SERVICE_NAME/-/_}
+serviceHost=$(find_env "${serviceName^^}_SERVICE_HOST")
+servicePort=$(find_env "${serviceName^^}_SERVICE_PORT_HTTP")
+
+update_config_map() {
+    # only execute the following lines if this container is running on OpenShift
+    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ]; then
+        local namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
+        local token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+        local response=$(curl -s --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
+            -H "Authorization: Bearer $token" \
+            -H 'Accept: application/json' \
+            -H 'Content-Type: application/strategic-merge-patch+json' \
+            -XPATCH \
+            https://${KUBERNETES_SERVICE_HOST:-kubernetes.default.svc}:${KUBERNETES_SERVICE_PORT:-443}/api/v1/namespaces/${namespace}/configmaps/myapp-kieserver -d ${1} )
+    fi
+}
+
+if [ "${serviceHost}x" != "x" -a "${servicePort}x" != "x" ]; then
+    auth="$(echo -n "${KIE_ADMIN_USER}:${KIE_ADMIN_PWD}" | base64)"
+    if [ ! -z "${KIE_SERVER_ID}" ]; then
```

Comment:
Could this be more simply expressed as:

if [ -n "${KIE_SERVER_ID}" ]; then

Reply:
agree simpler and result would be the same.